tactics/ind_tables.mli

```diff
@@ -30,7 +30,9 @@ type mutual_scheme_object_function =
 type individual_scheme_object_function =
   internal_flag -> inductive -> constr Evd.in_evar_universe_context * Evd.side_effects
 
-(** Main functions to register a scheme builder *)
+(** Main functions to register a scheme builder. Note these functions
+   are not safe to be used by plugins as their effects won't be undone
+   on backtracking *)
```

Comment:
This comment is not accurate: it's safe to use them, the only thing to be careful about is that they declare an object on the libstack.

Reply:
Why it is not accurate? The comment seems correct to me. `declare_scheme_object` does just add an entry into a hash table which is _not_ registered with the Summary nor with libstack, thus unsafe to use from plugins.

Reply:
OK.